Makefile

```diff
@@ -0,0 +1,19 @@
+all: install
+
+clone:
+	git clone git@github.com:pacificclimate/pycds.git
+
+install:
+	pip install -U pipenv
+	pipenv install --dev
+	pipenv install -e .
+
+local-pytest-image:
+	docker build -t pcic/pycds-local-pytest -f docker/local-pytest/Dockerfile .
+
+local-pytest-run:
+	py3clean .
+	docker run -it -v $(shell pwd):/codebase pcic/pycds-local-pytest
+
+test-db-image:
+	docker build -t pcic/pycds-test-db -f docker/test-db/Dockerfile .
```

Comment:
The `Makefile` looks great! Nice work. I'm curious, if you have access to the `Makefile` when would you make use of the `clone` target here?